contentprovider.go

```diff
@@ -267,6 +267,8 @@ const (
 	// TODO - how to scale this relative to rank?
 	scorePartialWordMatch   = 50.0
 	scoreWordMatch          = 500.0
+	scoreBase               = 7000
+	scorePartialBase        = 4000
```

Comment:
minor nit: match style of other lines
```suggestion
	scoreBase               = 7000.0
	scorePartialBase        = 4000.0
```